source/_includes/installation/container.md

```diff
@@ -32,7 +32,16 @@ Installation with Docker is straightforward. Adjust the following command so tha
   {% include installation/container/cli.md %}
 {% endif %}
 
-Once the Home Assistant Container is running Home Assistant should be accessible using `http://<host>:8123` (replace <host> with the hostname or IP of the system). You can continue with onboarding.
+Once the Home Assistant Container is running Home Assistant should be accessible using `http://<host>:8123` (replace <host> with the hostname or IP of the system). 
+
+If you are running with Docker Desktop for Windows or Mac, you will need to map port 8123 in the docker container to port 8123 of your host.  Do this by adding the following within the `homeassistant` section of your docker compose file:
```

Comment:
Hi there @NickDrouin 👋 

Unfortunately, this change is not correct.

Home Assistant requires the host network to function correctly. Docker Desktop on these operating systems doesn't support that. Hence, you need this additional port. However, that makes HA run in an unsupported way.

For that reason, you won't find Docker setup documentation for those operating system in our documentation.

Nevertheless, thank you for being willing to contribute 👍 


../Frenck